scripts/HFIR_4Circle_Reduction/FindUBUtility.py

```diff
@@ -10,8 +10,8 @@
 from __future__ import (absolute_import, division, print_function)
 import os
 
-from . import ui_AddUBPeaksDialog
-from . import ui_UBSelectPeaksDialog
+
```

Comment:
Up here (in files that require `load_ui`) add:
```python
try:
    from mantidqt.utils.qt import load_ui
    from mantidqt.utils.qt import load_ui
except ImportError:
    Logger('reasonablelogname').information('Using legacy ui importer')
    from mantidplot import load_ui
```
to use below. The second form will be removed once more parts of the qt common framework make it into the osx and windows packages.